Ziti.NET.Standard/src/OpenZiti/ZitiStatus.cs

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright NetFoundry Inc.
+Copyright 2019-2020 NetFoundry, Inc.
```

Comment:
this is the wrong copyright. please revert to "Copyright NetFoundry Inc.". thx

Reply:
done